javascript/javascript3/week3/homework/promise/main.js

```diff
@@ -0,0 +1,20 @@
+const div=document.createElement('div')
+async function fecthData(){
+  const response1=await  fetch('https://api.github.com/search/repositories?q=user:Vahablotfi')

```

Comment:
It's better to handle error when you fetch data from api

Reply:
`response` is not a good name, try to use something more meaningful like user or vahabUser